descriptions/AntiCheat.NEAC_Protect.md

```diff
@@ -0,0 +1 @@
+NEAC Protect is an anticheat solution developed by NetEase and used in Naraka: Bladepoint.
```

Comment:
For consistency with other descriptions, should use `anti-cheat`.

I'm don't think mentioning Naraka is necessary here.